packages/core/src/components/forms/fileInput.tsx

```diff
@@ -135,6 +139,14 @@ export class FileInput extends React.PureComponent<IFileInputProps, {}> {
         );
     }
 
+    private createSheet = () => {
```

Comment:
I don't want components to directly create new sheets like this... I think you can achieve the same result by conditionally adding a class name in the `render()` method... and style that class declaratively in `_file-input.scss`

Reply:
Thanks, I did too hard. Now everything seems fine